tests/unit/EmojiTest.js

```diff
@@ -78,14 +78,19 @@ describe('EmojiTest', () => {
         expect(EmojiUtils.containsOnlyEmojis('😄  👋')).toBe(true);
     });
 
-    it('replaces an emoji code with an emoji and a space', () => {
+    it('replaces an emoji code with an emoji and a space if addSpace is passed as true', () => {
         const text = 'Hi :smile:';
-        expect(EmojiUtils.replaceEmojis(text)).toBe('Hi 😄 ');
+        expect(EmojiUtils.replaceEmojis(text, true)).toBe('Hi 😄 ');
     });
 
-    it('will not add a space after the last emoji if there is text after it', () => {
+    it('will not add a space after the last emoji when there is text after it even if addSpace is passed as true', () => {
         const text = 'Hi :smile::wave:no space after last emoji';
-        expect(EmojiUtils.replaceEmojis(text)).toBe('Hi 😄👋no space after last emoji');
+        expect(EmojiUtils.replaceEmojis(text, true)).toBe('Hi 😄👋no space after last emoji');
```

Comment:
```suggestion
        expect(EmojiUtils.replaceEmojis(text)).toBe('Hi 😄👋no space after last emoji');
```
Let's keep this test as is. And add another one:

`expect(will not add a space after the last emoji when there is text after it on mobile)` which adds `true` in the call.